addons/website_slides/static/src/js/slides_course_quiz.js

```diff
@@ -44,6 +47,9 @@ odoo.define('website_slides.quiz', function (require) {
             });
             this.quiz = quiz_data || false;
             this.readonly = slide_data.readonly || false;
+            this.publicUser = session.is_website_user;
+            this.url = document.URL.replace('&', '%26');
```

Comment:
looks awefull :s 
What is you problem here ? You want to give that url as a param of another url ?

Reply:
Done, used encodeURIComponent instead